### PR TITLE
debian: add missing "make -C data/systemd clean"

### PR DESCRIPTION
**Pull request by mvo5**
Add missing debian/rules clean for the systemd input files to ensure they are not stale.

### CODE REVIEW

File: packaging/ubuntu-14.04/snapd.service [hunk 1]
```diff
@@ -6,6 +6,7 @@ Requires=snapd.socket
 ExecStart=/usr/lib/snapd/snapd
 EnvironmentFile=/etc/environment
 Restart=always
+Type=notify
```
Comment (zyga) [inline packaging/ubuntu-14.04/snapd.service]: Intentional?

Comment (Conan-Kudo) [inline packaging/ubuntu-14.04/snapd.service]: Intentional or not, it doesn't belong in this PR.

Comment (mvo5) [inline packaging/ubuntu-14.04/snapd.service]: Intentional but I agree, it should be a separate PR. I will remove the commit 231035f92d79969c143f81acbd0df00bd6289b76 from this PR and propose separately.

**Comment by niemeyer** [comment]
Tests were broken, btw, on an apt install operation. Restarted.
